js/src/jupyter-leaflet.js

```diff
@@ -4,6 +4,11 @@ var L = require('leaflet');
 require('leaflet-splitmap');
 require('leaflet-draw');
 require('leaflet.markercluster');
+Heat = require('./leaflet-heat.js');
+L.HeatLayer = (L.Layer ? L.Layer : L.Class).extend(Heat);
```

Comment:
Is this a compatibility thing between several versions of leaflet?

Reply:
I don't know, this comes from the original code:
https://github.com/Leaflet/Leaflet.heat/blob/gh-pages/src/HeatLayer.js#L3